index.js

```diff
@@ -7,22 +7,24 @@
 
 'use strict';
 
-var handlebars = require('handlebars');
 var helpers = require('handlebars-helpers');
+var handlebars = require('handlebars');
 
 // register built-in helpers
 exports.init = function (options, params) {
-  if (helpers && helpers.register) {
-    helpers.register(handlebars, options, params);
+  options = options || {};
+  if (options.handlebars) {
+    handlebars = options.handlebars;
   }
+  helpers(null, options);
```

Comment:
this should just be `helpers(options)`, otherwise the `options` aren't used in this version of `handlebars-helpers`